egs/swbd/s5c/local/run_asr_segmentation.sh

```diff
@@ -0,0 +1,145 @@
+#! /bin/bash
```

Comment:
I don't see any indication in a higher-level of run.sh where this fits into the larger picture, and I don't see any WERs.